src/System.CommandLine/Invocation/InvocationPipeline.cs

```diff
@@ -79,5 +126,17 @@ private static int DefaultExceptionHandler(Exception exception, CliConfiguration
             }
             return 1;
         }
+
+        private static int ReturnCodeForMissingAction(ParseResult parseResult)
+        {
+            if (parseResult.Errors.Any())
+            {
+                return 1;
+            }
+            else
+            {
+                return 0;
+            }
+        }
```

Comment:
We should avoid using LINQ in places where it can be done at relatively low cost (perf):

```suggestion
            => parseResult.Errors.Count > 0 ? 1 : 0;
```